app/views/comments/_comment.html.erb

```diff
@@ -1,7 +1,21 @@
 <% if comment && comment.user %>
-  <% if comment.depth < 3 && !should_be_hidden?(comment, @root_comment) %>
-    <details open>
-      <summary><span>&nbsp;</span></summary>
+  <% if comment.depth < 4 && !should_be_hidden?(comment, @root_comment) %>
```

Comment:
I wonder if we should move this hardcoded 4 into a constant, e.g. `MAX_COMMENT_DEPTH`. Then we don't need to worry about the value here and a few lines below getting out of sync.

Reply:
I think that makes a lot of sense. But not sure if I should be the person doing this - not because it's not part of my role but because I just don't know how and where to place it to make it the most efficient and configurable.. Because I also imagine this COULD be something we will eventually add to admin config...